### DISCUSSION TITLE
v6.0.0

### DISCUSSION THREAD

**Discussion by jinh0** [DDMAL/Neon2#960] [Announcements]
v6.0.0 of Neon brings a new dashboard/homepage and many new features, along with breaking changes.

## Breaking Changes
- Browsing and uploading files has been moved to `dashboard.html`, allowing `index.html` to be a more proper homepage for Neon. `dashboard.html` also includes a more user-friendly and capable file system, including drag-and-drop and uploading multiple files at once!
- Server structure has been reorganized, and files are served differently: in development, `/edit/005r_pre-toggle.jsonld` ➡️ `/editor.html?storage=005r_pre-toggle.mei`. 

## What's Changed
* ✨ 💄 Add visual cues that tell users if they're in Edit mode or in Insert mode. by @cadagong in https://github.com/DDMAL/Neon/pull/876
* 💄 Move error logs into a modal window (but left the option to export logs into a file) by @cadagong in https://github.com/DDMAL/Neon/pull/883
* 🐛 Fix divlines jumping to wrong staff lines. by @jinh0 in https://github.com/DDMAL/Neon/pull/882
* 🐛 Fix bug that was causing Neon to start in Insert mode instead of Edit mode by @cadagong in https://github.com/DDMAL/Neon/pull/887
* ✨ 💄 Add "Display All" option in Display Panel. Removed scroll behaviour for now by @cadagong in https://github.com/DDMAL/Neon/pull/889
* 🐛 Fix issue that was causing misleading visual cues regarding which mode user is in (Edit vs Insert) by @cadagong in https://github.com/DDMAL/Neon/pull/896
* 🐛♻️ Fix selection of multiple staves by cmd-click by @jinh0 in https://github.com/DDMAL/Neon/pull/897
* 🐛 Fix "Insert to nearest syllable" when no neume component exists by @jinh0 in https://github.com/DDMAL/Neon/pull/902
* 🐛 Fix bounding box positioning on inserting neume action by @chetbae in https://github.com/DDMAL/Neon/pull/847
* 🐛 Fix @follows/@precedes check with Array.find() by @jinh0 in https://github.com/DDMAL/Neon/pull/900
* 🐛 Fix highlight of bounding box when hovering over syllable texts by @jinh0 in https://github.com/DDMAL/Neon/pull/910
* **✨ Save and load settings from localStorage by @jinh0 in https://github.com/DDMAL/Neon/pull/907**
* ✨ Add sample folios selection to index page by @chetbae in https://github.com/DDMAL/Neon/pull/918
* 🐛 Disable out-of-bounds glyph placement by @jinh0 in https://github.com/DDMAL/Neon/pull/911
* ⚡️ Improve drag selection performance by @jinh0 in https://github.com/DDMAL/Neon/pull/923
* 🐛 Fix custos placement on rotated staves by @jinh0 in https://github.com/DDMAL/Neon/pull/926
* ✨ Let accidentals be visually positioned anywhere by @jinh0 in https://github.com/DDMAL/Neon/pull/925
* ✨ Add MEI action for removing out of bounds glyphs by @jinh0 in https://github.com/DDMAL/Neon/pull/922
* ✨ Add types to notifications by @jinh0 in https://github.com/DDMAL/Neon/pull/924
* 🐛 Fix inside-drag-selection calculation by @jinh0 in https://github.com/DDMAL/Neon/pull/928
* 🐛 Add staff out-of-bounds check by @jinh0 in https://github.com/DDMAL/Neon/pull/929
* 💄 add q/w/e/r/t/y to hotkey modal window by @chetbae in https://github.com/DDMAL/Neon/pull/934
* 🐛 ♻️ Improved selection options (and corresponding hotkey behavior) for all selection modes. by @cadagong in https://github.com/DDMAL/Neon/pull/904
* **🎉 New Neon Index Page by @chetbae in https://github.com/DDMAL/Neon/pull/893**
* 🐛 Fix highlight of staves after resize or rotate by @jinh0 in https://github.com/DDMAL/Neon/pull/909
* ✨ Add unsaved changes indicator / alert by @jinh0 in https://github.com/DDMAL/Neon/pull/921
* ✨ Add 'Insert into syllable' action button for Clefs by @chetbae in https://github.com/DDMAL/Neon/pull/930
* 🐛 Prevent cmd/ctrl+r from updating highlight by @jinh0 in https://github.com/DDMAL/Neon/pull/949
* **✅ Add end-to-end testing with Cypress by @jinh0 in https://github.com/DDMAL/Neon/pull/941**
* ✨ Add OOB check for resizing staves/bboxes by @jinh0 in https://github.com/DDMAL/Neon/pull/937
* ✨ Reselect glyphs after being dragged by @jinh0 in https://github.com/DDMAL/Neon/pull/954
* ✨ Add octave displacement for clefs by @jinh0 in https://github.com/DDMAL/Neon/pull/956

**Full Changelog**: https://github.com/DDMAL/Neon/compare/v5.0.0...v6.0.0

~ The Neon team @yinanazhou , @chetbae , @cadagong , and @jinh0 .

<hr /><em>This discussion was created from the release <a href='https://github.com/DDMAL/Neon/releases/tag/v6.0.0'>v6.0.0</a>.</em>

**Comment by fujinaga** [comment]
Congratulations to the team!

**Comment by annamorphism** [comment]
should the read-me be changed accordingly? It still says release 4.1.1 (which I think is also the last merge with the master branch-- c.f. https://github.com/DDMAL/Neon/issues/913 )

**Reply by jinh0** [reply]
It should be updated now!
